cmd/storage-consumer/main.go

```diff
@@ -98,11 +103,6 @@ func init() {
 		log.Error("invalid storage scheme, the scheme of upstream-uri must be file/s3/azblob/gcs")
 		os.Exit(1)
 	}
-
-	if len(configFile) == 0 {
```

Comment:
Why remove this check?

Reply:
When the config file is not provided, the default `ReplicaConfig` will be used. If the user is assured that the default configuration meet their need, the config file can be vacant.